example-config.yml

```diff
@@ -269,6 +269,14 @@ itinerary:
 #     defaultRouteTextColor: '000000'
 #     longNameSplitter: ' - '
 #     order: 2
+# Use this config to overwrite the  accessibility score gradation
+# map that ships with otp-ui
+#accessibilityScore:
+#    gradationMap:
+#        0.0:
+#            color: "#ffb5b9"
+#            text: 'Not Accessible'
```

Comment:
This should also support message ids for i18n. Message ids for each locale can be defined in this file (see what was done in #443 for the custom app menu entries).

Reply:
Resolved in bc19c7b